frontend/modelarchive/src/test/resources/models/noop_no_archive/noop/service.py

```diff
@@ -0,0 +1,111 @@
+
+
+"""
+NoopService defines a no operational model handler.
```

Comment:
@harshbafna Why are these other files included in this PR? It should only contain torch package version related items